src/MosaicViews.jl

```diff
@@ -335,7 +351,16 @@ function _padded_cat(imgs; fillvalue, dims)
     if length(unique(map(axes, imgs))) == 1
         return cat(imgs...; dims=dims)
     else
-        cat(paddedviews(fillvalue, imgs...)...; dims=dims)
+        if center
+            # TODO: ~1.5x slower than non-centered version
+            reduce(sym_paddedviews(fillvalue, imgs...)) do x, y
+                x = OffsetArray(x, 1 .- first.(axes(x)))
+                y = OffsetArray(y, 1 .- first.(axes(y)))
+                cat(x, y; dims=dims)
+            end
```

Comment:
Why doesn't `cat(sym_paddedviews(fillvalue, imgs...)...; dims=dims)` work?

Reply:
`cat` requires arrays to have the same axes, so I have to offset `PaddedView` back to common axes here.

P.S., the whole `_padded_cat` isn't optimized in both memory-allocation and type stability.

Reply:
Ah, this is `cat` not supporting arrays that start indexing at something other than 1.

Note this isn't guaranteed for `paddedviews` either.

Reply:
fixed in dae37ab

Reply:
There's a sense in which `cat` lives in the "array-as-list" world rather than "discretization-of-a-function-over-a-domain" world. Should the result of `cat` always start indexing at 1, no matter what the inputs are? I'm unsure what the right answer is, but it would solve problems of index-priority if so.

See `PinIndices` from https://github.com/JuliaArrays/CatIndices.jl for another take on the same problem.